app/models/concerns/with_financial_incentives.rb

```diff
@@ -0,0 +1,31 @@
+module WithFinancialIncentives
+  extend ActiveSupport::Concern
+
+  included do
+    def has_bursary?
+      dfe_subjects.any?(&:has_bursary?)
```

Comment:
should this be 

`bursary_amount.positive?` 

ie something that uses `dfe_subject_which_defines_financial_incentives`?

it just feels like they should be in aligned 

in case of human errors